srcs/parsing/lexer.c

```diff
@@ -88,10 +93,29 @@ void	get_command_tables(t_cmd_table *cmd_table, int nb_cmds, char **instructions
 		if (cmd_table[i].nb_redirs)
 		{
 			cmd_table[i].redirs = get_redirs(instructions[i], cmd_table[i].nb_redirs);
+			j = 0;
+			while (j < cmd_table[i].nb_redirs)
+			{
+				if (is_empty(cmd_table[i].redirs[j].arg))
+					error_and_exit(REDIR_NO_ARG);

```

Comment:
You shouldn't exit. If this happens we still need to be on the shell. It won't run any other instruction from that line, but we still need to be on the shell.